recipes/gwtc-tools/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "gwtc-tools" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gwtc_tools-{{ version }}.tar.gz
+  sha256: 6672e0a735f816ca7ee230ea92e641e02e5f614f062e6532eccb03f5137e3b42
+
+build:
+  entry_points:
+    - gwtc_update_pipeline_gevents = gwtc:gwtc_update_pipeline_gevents
+    - gwtc_diff = gwtc:gwtc_diff
+    - gwtc_get_gevent_coinc_files = gwtc:gwtc_get_gevent_coinc_files
+    - gwtc_create_from_query = gwtc:gwtc_create_from_query
```

The `gwtc_test` executable is missing from this list. I think this is responsible for the failure.

```suggestion
    - gwtc_update_pipeline_gevents = gwtc:gwtc_update_pipeline_gevents
    - gwtc_diff = gwtc:gwtc_diff
    - gwtc_get_gevent_coinc_files = gwtc:gwtc_get_gevent_coinc_files
    - gwtc_create_from_query = gwtc:gwtc_create_from_query
    - gwtc_test = gwtc:gwtc_test
```

---

@chadhanna I've implemented this suggestion in this PR: https://github.com/chadhanna/staged-recipes/pull/1